### ISSUE TITLE
Dagger

### ISSUE THREAD

**Issue by andreabernini** [DCLP/dclpxsltbox#136]
Cruces desperationis should be written in Pap. Colon. 28.11 (TM 64255) 
- recto 18 https://github.com/DCLP/idp.data/blob/dclp/DCLP/65/64255.xml#l94
- verso 15 https://github.com/DCLP/idp.data/blob/dclp/DCLP/65/64255.xml#l117
- verso 16 https://github.com/DCLP/idp.data/blob/dclp/DCLP/65/64255.xml#l118

but the XML of LitPapText,57 is not accepted by the system.

**Comment by andreabernini** [comment]
Used also in Pap. Colon. 28.4 (TM 65053)
- verso 6 https://github.com/DCLP/idp.data/blob/dclp/DCLP/66/65053.xml#l113 
- verso 8 https://github.com/DCLP/idp.data/blob/dclp/DCLP/66/65053.xml#l115
- verso 9 https://github.com/DCLP/idp.data/blob/dclp/DCLP/66/65053.xml#l116

**Comment by paregorios** [comment]
@andreabernini please advise on the following:

1. What is the significance of the "cruces desperationis", i.e., what does it signify to the reader?
2. When you say "the XML of LitPapText,57 is not accepted by the system", what do you mean? What system? What error message are you getting?

Thanks.

**Comment by HolgerEssler** [comment]
Andrea is referring to my suggestion in our LitPapText spreadsheet line
57
(https://docs.google.com/spreadsheets/d/1F6_Du8DIjixRD4mnvfpbkcn3E0H2_SfBIztCO6Vl6gQ/edit#gid=1879498412).
The xml I proposed there has not yet been implemented.

**Comment by rla2118** [comment]
Tom, is this clear now?  The dagger is used to signal a corrupt passage in
the original manuscript.

**Comment by paregorios** [comment]
I now understand the significance of the symbols. I could still use clarification on what was done and what happened that led @andreabernini to write "the XML of LitPapText,57 is not accepted by the system".

**Comment by paregorios** [comment]
Or perhaps I'm now misunderstanding @HolgerEssler when he says "the xml I proposed there has not yet been implemented." Do we mean one of more of the following?:

1. The XSugar grammar has not yet been modified to produce the corresponding XML
2. The desired XML is not allowed by the current EpiDoc schema
3. There are not yet any files in idp.data/DCLP which the corresponding XML has been entered.
4. The XSLT files are not properly handling the XML in particular files.
5. @andreabernini is trying to enter this as part of a larger text into the PE and for some other reason it's erroring off.
6. Something else.

I'm trying to figure out if this is related to any other tickets and/or/also what the next steps for me should be.

**Comment by HolgerEssler** [comment]
I should have written:
1) The XSugar grammar has not yet been modified to produce the
corresponding XML.
2) The desired XML is not allowed by the current EpiDoc schema.
3) There are not yet any files in idp.data/DCLP which the corresponding
XML has been entered.
4) The XSLT files are not handling the proposed XML in the desired way

Example from Sappho, fr. 2
Display should look like:
†ττριννοις† ἄνθεσιν

Proposed XML
<app><orig cert="corruption">ττριννοις</orig></app> ἄνθεσιν

Proposed Leiden+
_†ττριννοις†_ ἄνθεσιν


There is however a problem:
Normally the dagger surrounds the passage (word or group of words) which
is corrupted and in these cases the above way works fine.
However, sometimes editors use a singe dagger to indicate that there is
some problem which cannot be narrowed down to a specific word. In this
case we would just have something like:
 τρεις † ἄνθεσιν
Should we double these single daggers, i.e. have
Leiden+
 τρεις _††_ ἄνθεσιν and
XML
<app>τρεις<orig cert="corruption"></orig></app> ἄνθεσιν ?
 Otherwise we would have to introduce also an empty element.

**Comment by paregorios** [comment]
OK, I think we should loft the proposed XML encoding on the Markup list to see if anybody else in the EpiDoc community has an opinion. Shall I do that?

**Comment by HolgerEssler** [comment]
That would be great. Mine was just a rough suggestion and I would be
very pleased, if others came up with something more suitable and more
sophisticated.

**Comment by paregorios** [comment]
OK will do.

**Comment by HolgerEssler** [comment]
#96 (https://github.com/DCLP/dclpxsltbox/issues/96) can be closed as doubling this issue.

**Comment by paregorios** [comment]
Tasks, as I see them in order:

 - [ ] Determine appropriate TEI/EpiDoc encoding for this and verify that it validates.
 - [ ] Get one or more texts in idp.data that conform to the XML encoding expectations so we can test.
 - [ ] Test and modify XSLT as necessary to produce desired HTML display of the markup.
 - [ ] Determine appropriate Leiden+ syntax and implement in XSugar

**Comment by paregorios** [comment]
@leoba @hcayless @gabrielbodard @wsalesky have any of you seen appropriate markup for this sort of thing in digitzed manuscripts?

**Comment by leoba** [comment]
Noting that @HolgerEssler's suggestion is:

`<app><orig cert="corruption">ττριννοις</orig></app>` 

To display as:

†ττριννοις† ἄνθεσιν Sappho, fr. 2 (with om.)	

I don't think `@cert` is right here - it's supposed to describe a degree of certainty the transcriber has in their transcription (usually high, medium, or low). Here, I think, we want to say something about the reading itself - the reading is corrupt in the manuscript, so we're noting the reading (about which we are certain), but we want to note that the reading is corrupted. Unfortunately I don't see another attribute that really suits.

On the element side I think `sic` would be a better choice than `orig`: "contains text reproduced although apparently incorrect or inaccurate" (http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-sic.html) - although `orig` is probably fine, this could just be my preference. 

The attribute is the kicker. I'd want to use something like `@type` but that would involve customizing the TEI, as neither `orig` nor `sic` are typed.

**Comment by HolgerEssler** [comment]
I agree with @leoba that `@cert` would be wrong and that `<sic>` is preferable. Since we are currently using `<sic>` only inside `<choice>` (and paired with `<corr>`) to mark mistakes that have been corrected, I am wondering whether we could simply use it on its own without any attribute (thus avoiding custumizing TEI). The logic for XSLT would be:
a) `<sic>` as child of `<choice>` is rendered as before
b) if `<sic>` is not a child of `<choice>` (i.e. we are dealing with an error that has not been corrected), display daggers at the position of  `<sic>` and `</sic>` (and perhaps even including the empty element `<sic/>`, if we want to have the possibility to handle a single dagger as well - which I would prefer).

**Comment by leoba** [comment]
This makes sense to me.

**Comment by paregorios** [comment]
Thanks all. I'll undertake to check on that encoding and whether it's spoken for for any other purpose elsewhere in EpiDoc and then we can move forward.

**Comment by paregorios** [comment]
This is for me to address so we can decide whether to move forward or to defer.

**Comment by hcayless** [comment]
FYI plain old `<sic>` is how we handle cruces in the DLL guidelines, so @leoba's suggestion is totally consistent with practice there.

**Comment by paregorios** [comment]
It is NOT the case, as asserted above, that ```<sic>``` is not valid vs. the EpiDoc schema. I have just tested it and it is fine.

**Comment by paregorios** [comment]
There are currently no examples of ```//t:sic[not(ancestor::t:choice)]``` anywhere in DCLP or DDB; therefore, code changes cannot be tested. Given time remaining, I'm moving this item to post-release.

**Comment by HolgerEssler** [comment]
This is only slightly related, but when looking for the example mentioned above, i.e. http://litpap.info/dclp/62716, I could not find the text. However, I am fairly sure that @rla2118 did encode this one (the Sappho ostracon) and I am wondering where it ended up. I suppose with this text we would also have at least one example.

**Comment by paregorios** [comment]
I have no idea how to try to run this down. @jcowey ?
